SalesforceCommerceCloud/pwa-kit
# Performance: lazy load popover

# Description



We want to lazy load the Chakra Popovers because they are big CPU eaters.

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- lazyload popover

# How to Test-Drive This PR

- `npm start` and make sure the popover still works, the account icon on header uses the component

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
